server/webhook_parser.go

```diff
@@ -74,12 +74,8 @@ func ParseWebhook(bb []byte) (wh Webhook, err error) {
 		default:
 			wh, err = parseWebhookUnspecified(jwh)
 		}
-	case commentCreated:
-		wh, err = parseWebhookCommentCreated(jwh)
-	case commentUpdated:
-		wh, err = parseWebhookCommentUpdated(jwh)
-	case commentDeleted:
-		wh, err = parseWebhookCommentDeleted(jwh)
+	case commentCreated, commentUpdated, commentDeleted:
```

Comment:
Let's ensure in testing that this change doesn't have any unexpected side affects.